views/healthcheck/index.js

```diff
@@ -13,5 +15,22 @@ module.exports = view.extend({
         back: '/profile',
         config: config,
         package: packageJSON
+    },
+    computed: {
+      options: function() {
+        return i18n.supportedLangs();
+      },
+      currentLang: function() {
```

Comment:
I would just set this on the `data` object once instead, so you can use this value as the model of the select element